_episodes/02-vcs.md

```diff
@@ -280,21 +274,21 @@ commands to have `git-annex` branch merged correctly.
 > {: .solution}
 {: .challenge}
 
-> ## How to add file a.txt directly under git, and file b.dat under git-annex?
-> ### Simple (one time)
-> Use `git add` for adding files under git, and `git annex add` to
+> ## How can we add the file a.txt directly under git, and file b.dat under git-annex?
+> ### Simple method (initial instance)
```

Comment:
I am not sure if `initial instance` a good name here... not sure what it means... at least may be `initial invocation` or `once` .